examples/tfengine/generated/org_foundation/devops/main.tf

```diff
@@ -56,16 +63,84 @@ module "state_bucket" {
   location   = var.storage_location
 }
 
+# Devops project owners group.
+module "owners_group" {
+  source  = "terraform-google-modules/group/google"
+  version = "~> 0.2"
+
+  count = var.project.owners_group.exists ? 1 : 0
```

Comment:
the logic is opposite.

count should 1 when the group does not exist

Reply:
That's right. Thanks!